app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -0,0 +1,318 @@
+
+* [India Young Professionals Scheme visa](https://www.gov.uk/india-young-professionals-scheme-visa) 
+
+* [Family visa](https://www.gov.uk/uk-family-visa) 
+
+* [UK Ancestry visa](https://www.gov.uk/ancestry-visa)  
+
+* [British National (Overseas) visa](https://www.gov.uk/british-national-overseas-bno-visa) 
+
+* [High Potential Individual visa](https://www.gov.uk/high-potential-individual-visa)  
+
+For the following Ukrainian and Afghan visas, you can apply for your visa from within the UK:  
+
+* [Afghan citizens resettlement scheme](https://www.gov.uk/guidance/afghan-citizens-resettlement-scheme) 
+
+* [Afghan Relocations and Assistance Policy](https://www.gov.uk/government/publications/afghan-relocations-and-assistance-policy/afghan-relocations-and-assistance-policy-information-and-guidance) 
+
+* [Ukraine Family Scheme visa](https://www.gov.uk/guidance/apply-for-a-ukraine-family-scheme-visa) 
+
+* [Ukraine Sponsorship Scheme](https://www.gov.uk/guidance/apply-for-a-visa-under-the-ukraine-sponsorship-scheme) 
+
+### 3. Date of arrival
+
+You must move to England no more than 3 months before the start of your course.   
+
+### Non-eligibility for trainee teachers 
+
+You will not be eligible for the international relocation payment if you are: 
+
+* a UK citizen 
+
+* an Irish citizen 
+
+* a non-UK citizen who moved to England to live permanently more than 3 months before the start of your course 
+
+* doing an assessment only QTS 
+
+* training to teach only English language and literature
+
+## How trainee teachers can get the international relocation payment 
+
+### Fee-paying trainees 
+
+If you have a place on a teacher training course where you pay a fee, you do not need to apply for the international relocation payment (IRP). Your training provider will email you if your choice of course makes you eligible for the IRP. Your visa status and date of arrival will be checked by the Department for Education. If you are eligible, you’ll get the IRP around the end of your first term. 
+
+If you believe you meet the eligibility requirements, but have not been informed by your teacher training provider that you may be awarded the IRP, please contact your training provider directly first. If your training provider cannot help you, you can email us at teach.inengland@education.gov.uk. 
+
+### Salaried trainees 
+
+If you have a place on a teacher training course where you are paid a salary to work while you train, email us at IRP.ExpressInterest@education.gov.uk to let us know you are interested in applying for the IRP. You will be asked to fill in a form giving us your details and we’ll get in touch when IRP applications open for salaried trainees in autumn 2023. If you are eligible, you’ll get the IRP around the end of your first term. 
+
+## The international relocation payment for teachers  
+
+To be eligible for the international relocation payment (IRP), you must meet all 3 of the following requirements.
+
+### 1. Teaching subject
```

Comment:
Is it good practice to have 2 headings with the same text? (I'm thinking this is probably bad for accessibility, and not useful for users trying to scan through to find the relevant information for them)

Reply:
yes - it is not good practice in terms of accessibility to have duplicate headings on a single page

Reply:
I have relabelled them...